packages/DatePicker/DatePicker.jsx

```diff
@@ -0,0 +1,138 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import momentPropTypes from 'react-moment-proptypes'
+import moment from 'moment'
+import isBeforeDay from 'react-dates/lib/utils/isBeforeDay'
+
+import 'react-dates/initialize'
+import SingleDatePicker from 'react-dates/lib/components/SingleDatePicker'
+import DayPickerSingleDateController from 'react-dates/lib/components/DayPickerSingleDateController'
+import 'react-dates/lib/css/_datepicker.css'
+
+import { safeRest, getCopy } from '@tds/util-helpers'
+
+import DecorativeIcon from '@tds/core-decorative-icon'
+
+import { CalendarContainer, LabelText, HiddenInputFieldContainer } from './styles'
+
+import copyDictionary from './datePickerText'
+/**
+ * The `DatePicker` component is used to select a single date. It is available as an inline date picker or overlay date picker where the customer may select a date, either by keying in (input form field) or selecting through the overlay.
+ * @version ./package.json
+ */
+
+class DatePicker extends Component {
+  state = {
```

Comment:
Since there's only one state, this can be a hook and the component can be made into a functional one.